workers/unity/Packages/com.improbable.gdk.mobile/Editor/LaunchMenu.cs

```diff
@@ -1,21 +1,27 @@
+using System.Diagnostics;
 using System.IO;
+using System.Linq;
 using System.Text;
 using Improbable.Gdk.Core;
 using Improbable.Gdk.Tools;
 using UnityEditor;
 using UnityEngine;
+using Debug = UnityEngine.Debug;
```

Comment:
I assume that happened because of the System.Diagnostics import, shouldn't be necessary to define Debug